### PR TITLE
Luw 28 Routing

### PR DESCRIPTION
**Pull request by fannymartensson**
- Most of the changes comes from LUW-27. 
- Without globalstate for LoadingScreen. 
- Will setup task for LoadingScreen.

### CODE REVIEW

File: src/pages/TeamPage/TeamPage.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import useStyles from '../styling/style';
+
+export const TeamPage = () => {
+  const classes = useStyles();
+  const [team, setTeam] = useState([]);
+
+  const fetchAndSetTeam = async () => {
+    try {
+      const { data } = await axios.get('/api/team-member');
+      setTeam(data as any);
```
Comment (boomboxbacardi) [inline src/pages/TeamPage/TeamPage.tsx:18]: Don't we have the type for team?

File: src/routing/Routing.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { BrowserRouter, Routes, Route, Outlet } from 'react-router-dom';
+
+import { HomePage } from '../pages/HomePage';
+import { TeamPage } from '../pages/TeamPage';
+import { AboutPage } from '../pages/AboutPage';
```
Comment (boomboxbacardi) [inline src/routing/Routing.tsx:5]: Should be imported from index file instead.

`import { HomePage, TeamPage, AboutPage } from '../pages`

**Review by boomboxbacardi — APPROVED** [review]

File: src/pages/TeamPage/TeamPage.tsx (hunk 1 above)
Comment (fannymartensson) [reply src/pages/TeamPage/TeamPage.tsx:18]: We can't use team type, because we fetch the entire object with keys etc. Should we add a new type for this?

Comment (boomboxbacardi) [reply src/pages/TeamPage/TeamPage.tsx:18]: Alright. Yeah, we walk to type it out

**Pull request merged**